openlibrary/plugins/upstream/addbook.py

```diff
@@ -505,7 +531,7 @@ def process_input(self, i):
         else:
             edition = None
 
-        if 'work' in i and self.do_work_keys_match(i):
+        if 'work' in i and self.is_work_data_important(i):
```

Comment:
I'm not sure I like the new name better than the old. The old describes 95% of the cases. If we're going to rename, I think something better than "important" is needed. `use_work_edits`? `are_work_edits_valid`?

Reply:
I like `use_work_edits`